data-mermaid/mermaid-webapp
# Remove reference link updated

[trello card](https://trello.com/c/9ZqgZ8Hw/939-remove-reference-icon-from-benthic-attribute-fish-fields): will update commit link after merged in

- removed for benthic lit, benthic pit, benthic photo quadrat, and fish belt observation fields
- also removed test for reference link

note:
I left `REACT_APP_MERMAID_REFERENCE_LINK` in the `yaml` files and `env` as I see its still being used in the header nav link. Added comment in trello card to confirm with Kim

to test:
1. Go to observation tables in benthic pit, benthic lit, benthic photo quadrat, colonies bleached, and fish belt 
2. add an observation row
3. add a selection
4. ensure reference link does not appear in row
